logging-view-plugin-resources.yml

```diff
@@ -78,10 +78,19 @@ spec:
       containers:
         - name: logging-view-plugin
           image: 'quay.io/gbernal/logging-view-plugin:latest'
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
           ports:
             - containerPort: 9443
               protocol: TCP
           imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: 10m
+              memory: 50Mi
```

Comment:
Really that low? What's the consideration? Because we serve static files from nginx only?

Reply:
yes, we are not doing any other backend work now (might change in the future)

Reply:
👍🏽